src/query/generated/proto/rpc/query.proto

```diff
@@ -1,9 +1,10 @@
 syntax = "proto3";
 
-package rpcpb;
+package rpc;
```

Comment:
Yeah so the reason why I updated the name of this is so `make test-all-gen` would pass. Prateek was working on a fix I think? cc @prateek

Reply:
Yeah noticed it was failing :\ unfortunately this needs to match the package name on the other side as far as I can tell